src/impl/cli_ui.ts

```diff
@@ -89,4 +89,8 @@ export class CliUi extends GameUi {
   displayText(text: string): void {
     this.interface.write(text);
   }
+
+  displayWrongInputMessage(): void {
+    this.displayText("Please choose a right option.\n")
```

Comment:
@willypuzzle Just a knit pick, but I would suggest we change the error message to "Please select a valid option" As there are no right answers, just valid ones. What do you think?

Reply:
Yes, you're right!

Reply:
@nombrekeff I did it!

Reply:
Great!! I will wait a bit to merge this in case someone else wants to check the original code! won't take much to merge, maybe tomorrow morning or so! Thanks again for taking the time to help out :)

Reply:
Ok, no problem, merge when you wish :)